appmap/flask.py

```diff
@@ -123,6 +125,12 @@ def after_request(self, response):
         return response
 
 
+@patch_class(jinja2.Template)
+class TemplateHandler(BaseTemplateHandler):
+    # pylint: disable=missing-class-docstring, too-few-public-methods
+    pass
```

Comment:
I thought I understood what this was doing, but seeing the `pass` here, I guess I don't.

Some of my confusion comes from having this class be named TemplateHandler, i.e. the same name that's renamed on import from `web_framework`.

I the other thing I'm confused about is whether there's a method named `render` in the `Template` class in both django and jinja2. Is there?

Can you add some comments to expand on what's happening, please?

Reply:
> Some of my confusion comes from having this class be named TemplateHandler, i.e. the same name that's renamed on import from `web_framework`.

I've seen this idiom in other libraries, it seems common in python. Ie. have a `lib.base.Widget` and then `lib.specific.Widget` inheriting from the former; both these names are already unambiguous and not really intended to be used together, so there's no need to make them redundant as `lib.base.BaseWidget` and `lib.specific.SpecificWidget`. Renaming `lib.base.Widget` on import to `BaseWidget` simply avoids the need to spell out `lib.base.Widget` in full in the class definition.

> I the other thing I'm confused about is whether there's a method named `render` in the `Template` class in both django and jinja2. Is there?

Yes.

> Can you add some comments to expand on what's happening, please?

Ok, I'll try :)

Reply:
> I thought I understood what this was doing, but seeing the `pass` here, I guess I don't.

I suppose I might as well have written simply
```python
patch_class(jinja2.Template)(appmap.web_framework.TemplateHandler)
```
instead of defining a decorated empty class. Would that be any clearer?

Reply:
Ok re the rename on import. I'll get used to it, I guess. ;)

The comment you added does a good job of clarifying what's going on. With it, I think the empty class is better than the one-liner.